colossus-tests/src/test/scala/colossus/core/ConnectionHandlerSpec.scala

```diff
@@ -66,14 +67,20 @@ class ConnectionHandlerSpec extends ColossusSpec {
       val probe = TestProbe()
       class MyHandler extends BasicSyncHandler {
         override def onUnbind() {
+          println("UNBINDING!!!!!!!")
```

Comment:
we can probably remove this println.  On the other hand..we REALLY need to start getting our logging situation sorted out.